scripts/package.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
I think I would do it without `package.json` and `yarn.lock`.

Reply:
And just a `yarn install yaml` command?

Reply:
I would be OK installing all the dependencies, we can have the workflow share caches for `node_modules`.

Reply:
But doesn't the main `yarn install` happen inside docker when building the docker image? How easy would it be to share caches for that?

Reply:
I guess we could do the yarn install outside the image, and then COPY the node_modules in

Reply:
It would not work when running with `image: "Dockerfile"` with the `COPY` of `node_modules`.

Reply:
I've updated -- it seems to work (I'm yarn install'ing in both publish task & run-all-tests task)